### DISCUSSION TITLE
Tło do kart

### DISCUSSION THREAD

**Discussion by mattyl006** [emkarcinos/WMIAdventure#418] [Design]
@emkarcinos możesz spróbować jako te tło do kart zrobić jakieś kody binarne estetycznie, albo jakiś kod z programowania

**Comment by emkarcinos** [comment]
@mattyl006 Tak, właśnie o takim motywie mniej więcej myślałem. Cały czas staram się robić burzę mózgu wokoło informatycznych tematów
